docs/layers/identity/aws-sso.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: "AWS Identity Center (SSO) ClickOps"
-sidebar_label: "Setup Identity Center"
+sidebar_label: "AWS SSO"
```

Comment:
IMO, the sidebar labels that have instructions should have a verb.

Reply:
reverted